### ISSUE TITLE
[Applied anywhere in the UAT site] (other + specify) to be removed from the individual contact card

### ISSUE THREAD

**Issue by hohangleung** [MSFREACH/MSF-REACH-COLLABORATION#122]
- [x] Once clicked 'other', a box free text field should appear with the word 'Specify'. This will be added to the list of options of the dropdown for the next user to be able to choose it.

**Comment by hohangleung** [comment]
That is added from day one, is it needed to be removed?

**Comment by LucieGueuning** [comment]
Yes, the wording of the drop down is 'Other'. And then a box appear for the specification. It should NOT be 'Other (other + specify).

**Comment by hohangleung** [comment]
![Kapture 2020-01-16 at 15 25 13](https://user-images.githubusercontent.com/50646609/72507305-9aa8a600-387e-11ea-9b5e-d9636744344f.gif)
![Kapture 2020-01-16 at 15 26 34](https://user-images.githubusercontent.com/50646609/72507306-9b413c80-387e-11ea-8e60-f9911d879f3e.gif)

Updated, please check

**Comment by LucieGueuning** [comment]
Once clicked 'other', a box free text field should appear with the word 'Specify'. This will be added to the list of options of the dropdown for the next user to be able to choose it.

**Comment by hohangleung** [comment]
![other_specify](https://user-images.githubusercontent.com/50646609/78509723-cf176880-77c2-11ea-9eb8-66deb394fcb5.gif)

The "Other + Specify" box has been added.

There are some details on this feature I would like to clarify.
- When you save the form (Contacts, as example), the inputted text will also get saved to the list
- This new option will NOT removed even after the contact card deleted
- The next user will get auto-suggestion when they use the "Other + Specify" box
- The user can select from the auto-suggestion box first, then edit the text (Will save as new option)

Please let me know if it's what you looking for.

**Comment by LucieGueuning** [comment]
It is all correct, except that:
- The new option will be added to the original options list of drop down (in your example, in the left boxe). 
- New user can select that new option without the need to go to other+ specify 
- And the new options need to be confirm and able to correct spelling mistake by admin.

**Comment by hohangleung** [comment]
Event and Contact Applied
